distributed/tests/test_executor.py

```diff
@@ -3576,3 +3576,25 @@ def test_idempotence(s, a, b):
 
     yield e._shutdown()
     yield f._shutdown()
+
+
+@gen_cluster(executor=True)
+def test_make_serializable(e, s, a, b):
+    from distributed.executor import serialize_for_publish, deserialize_published
+    import pickle
+    f = delayed(lambda: 1)()
+    futures = yield e._scatter([f, f, f])
+    with pytest.raises(AttributeError):
+        pickle.dumps(futures)
+    with serialize_for_publish(futures) as f:
+        d = pickle.dumps(f)
+    assert d
+
+
+@gen_cluster(executor=True)
+def test_publish_simple(e, s, a, b):
+    data = yield e._scatter(range(12))
+    e.publish_dataset(data, 'data')
+    yield gen.sleep(0.2)
+
+    assert s.published_data
```

Comment:
OK, but that's for the next test - this one just says that the data made it to the scheduler. Getting it back is another matter!
As an aside, start=True would create a blocking version of the Executor?